src/neuroconv/tools/hdmf.py

```diff
@@ -114,13 +119,14 @@ def estimate_default_buffer_shape(
                 k3 = math.floor(target_buffer_bytes / buffer_bytes)
                 padded_buffer_shape[axis] *= k3
                 break
+        padded_buffer_shape = tuple(int(value) for value in padded_buffer_shape)  # To avoid overflow from math.prod
```

Comment:
Does this comment corresopnding to casting the elements to int?

Reply:
It corresponds to converting the numpy array of numpy ints to a tuple of python ints. Otherwise it's another potential source of the overflow

Reply:
Got you I guess we could do it from line 109 already?

Reply:
Needs to be an array for line 125

Doesn't need to be an array on exiting the conditions after 126

Reply:
Got you, I looked for numpy operations and did not see that implicit one : )